src/core/pdf_manager.js

```diff
@@ -110,11 +114,11 @@ var BasePdfManager = (function BasePdfManagerClosure() {
 })();
 
 var LocalPdfManager = (function LocalPdfManagerClosure() {
-  function LocalPdfManager(docId, data, password, evaluatorOptions) {
+  function LocalPdfManager(docId, data, password, evaluatorOptions, filename) {
     this._docId = docId;
     this.evaluatorOptions = evaluatorOptions;
     var stream = new Stream(data);
-    this.pdfDocument = new PDFDocument(this, stream, password);
+    this.pdfDocument = new PDFDocument(this, stream, password, filename);
```

Comment:
Undo (Use `this._filename = filename;` right after the `_docId` assignment above).